### PR TITLE
GP2-234-Informational-Message

### PR DESCRIPTION
**Pull request by csbarbosa**
(Ticket: https://asaphub.atlassian.net/browse/GP2-234)
Change color and format of the information note in resources

### CODE REVIEW

File: apps/storybook/src/gp2-components/Resources.stories.tsx [hunk 1]
```diff
@@ -15,10 +15,15 @@ const getResources = (length = 1): gp2.Resource[] =>
...
 const props = {
-  headline: `Please note, this is a private space for this working group on the
-        network. Nobody outside of this working group can see anything that you
-        upload here.`,
-
+  headline: (
+    <>
+      <b>Please note</b>
+      <br />
+      Please note, this is a private space for this working group on the
```
Comment (dianaguimaraes) [inline apps/storybook/src/gp2-components/Resources.stories.tsx:22]: I believe this copy is not correct and you're repeating `Please note`

File: packages/gp2-components/src/templates/WorkingGroupResources.tsx [hunk 2]
```diff
@@ -6,9 +6,15 @@ type WorkingGroupResourcesProps = Omit<
...
 >;
 
-const headline = `Please note, this is a private space for this working group on the
-        network. Nobody outside of this working group can see anything that you
-        upload here.`;
+const headline = (
+  <div>
+    <b>Please note</b>
+    <br />
+    Please note, this is a private space for this working group on the network.
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/templates/WorkingGroupResources.tsx:13]: Same

File: packages/gp2-components/src/organisms/Resources.tsx [hunk 3]
```diff
@@ -15,14 +16,15 @@ import {
...
+import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
 import { useState } from 'react';
 import { addIcon, editIcon } from '../icons';
 import { mobileQuery, nonMobileQuery } from '../layout';
 import colors from '../templates/colors';
 
 export type ResourcesProps = {
   resources?: gp2.Resource[];
-  headline: string;
+  headline: EmotionJSX.Element;
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/organisms/Resources.tsx:27]: Wondering if this the right type to use? 🤔

File: packages/react-components/src/atoms/Card.tsx [hunk 4]
```diff
@@ -45,6 +46,11 @@ export const accents: Record<AccentVariant, CSSObject> = {
     borderColor: colors.steel.rgb,
     boxShadow: `0px 2px 4px ${colors.steel.rgb}`,
   },
+  warning100: {
```
Comment (jhonny-m) [inline packages/react-components/src/atoms/Card.tsx:49]: I would just name it warning

**Review by jhonny-m — APPROVED** [review]
Left a comment, but after addressing it you can merge it

**Pull request merged**